lib/tasks/data-key.rake

```diff
@@ -8,4 +8,63 @@ namespace :"data-key" do
     key = Base64.strict_encode64(Slosilo::Symmetric.new.random_key)
     print key
   end
+
+  desc "Validates the value of ENV['CONJUR_DATA_KEY']"
+  # This task validate the master data key which is stored in `ENV['CONJUR_DATA_KEY']`
```

Comment:
validate**s** the (master?) data key **that** is stored..."

Reply:
Again, the master key is something else. Here we refer to the data key.